### ISSUE TITLE
global view and local view

### ISSUE THREAD

**Issue by Backdrop9019** [kahnchana/svt#16]
In the paper, it is written that the global view uses T(8,16) and the local view uses (2,4,8,16). However, in the published code, it is simply fixed to 8 frames. 

Should I use rand_fr as True? However, global is still used as (4,8).

`        if rand_fr:
            global_1 = temporal_sampling(frames, 0, max_len - 5, 4)
            global_2 = temporal_sampling(frames, 5, max_len, 8)`
 Can you please disclose the exact config that I can reproduce?

**Comment by kahnchana** [comment]
Yes - it should be set to `True` for that to work. 

The current version of code may use a smaller frame count for more efficient training (which will decrease performance a bit).

**Comment by Backdrop9019** [comment]
Would changing the global view to 8 16 yield the same performance as the paper suggests?

**Comment by kahnchana** [comment]
Yes, the values reported in the paper use that setting (which was our optimal setting to get best results).

**Comment by kffeng** [comment]
@kahnchana Hi, thank you for providing the code. Did you set the learning rate as lr=5e-4 * (8 * 4) / 256 and epochs as 20 during the pre-training phase?

**Comment by kahnchana** [comment]
Yes - the values in the current code (set by config files) are the ones we used.

**Comment by kffeng** [comment]
@kahnchana Hi, I would like to know if the open-source weight file you provided was obtained when rand_fr=true. Why is it that when I set rand_fr=false and the learning rate to lr=5e-4 * (8 * 4) / 256 (on 8 gpu, with a batch_size of 4 per gpu), and epochs as 20 during the pre-training, the pre-training weights I obtained only achieved a top1 result of 74% on K400 when using slowfast for inference,lr=5e-3 and epochs as 15 during the finetune .When I use your weights to run finetune, I can achieve a result of 77.3%. Did I miss something?

**Comment by kffeng** [comment]
@kahnchana Hi, I set `rand_fr=True` and changed the global view to 8 16. So, in `self.time_embed = nn.Parameter(torch.zeros(1, num_frames, embed_dim))`, is `num_frames=8 or 16`? If it is 16, should I set `slow frames=8 or 16` during the slowfast finetuning stage? I hope to receive your reply. Thank you very much.
